data/nvidia/1.23.0/kmod-5.15-nvidia.toml

```diff
@@ -0,0 +1,47 @@
+# Extracted from https://github.com/bottlerocket-os/bottlerocket/tree/develop/packages/kmod-5.15-nvidia/Cargo.toml
```

Comment:
This isn't really true anymore can we update it for the core kit.
(Same comment for the other 2 versions)

Reply:
This change is introduced by one of the scripts that pulls information about the nvidia drivers into the website. I will fix the script and these comments in a separate PR. It doesn't affect the renders in the website.